client/components/theme-collection/index.tsx

```diff
@@ -65,19 +65,19 @@ export default function ThemeCollection( {
 							prevEl: '.theme-collection__carousel-nav-button--previous',
 						},
 						threshold: 5,
-						slideToClickedSlide: true,
+						slideToClickedSlide: false,
+						rewind: true,
 						slidesPerView: 1.2,
 						breakpoints: {
-							// break-small in Gutenberg breakpoints
-							'600': {
+							// deprecated Calypso breakpoints used in the Theme Showcase
```

Comment:
Gutenberg has a break-small breakpoint, I don't understand why we need to change this to stating that Calypso has this breakpoint deprecated when Gutenberg has this breakpoint and it's not deprecated there.

Reply:
I agree with the principle, but LoTS (not sure about LiTS) uses the old 660 breakpoint to reduce the horizontal padding from 32px to 16px.
Using a different breakpoint would mean having an unnecessary UI jump between 660 and 600.